packages/devtools_app/lib/src/screens/profiler/cpu_profile_service.dart

```diff
@@ -13,11 +13,10 @@ extension CpuProfilerExtension on VmServiceWrapper {
     required int startMicros,
     required int extentMicros,
   }) async {
-    return await serviceManager.service!.getCpuProfileTimeline(
-      serviceManager.isolateManager.selectedIsolate.value!.id!,
-      startMicros,
-      extentMicros,
-    );
+    final isolateId = serviceManager.isolateManager.selectedIsolate.value!.id!;
+    final cpuSamples = await serviceManager.service!
+        .getCpuSamples(isolateId, startMicros, extentMicros);
```

Comment:
nit: trailing comma afer 'extendMicros'. I'm surprised the linter didn't catch this - have you merged master into your PR recently?

Reply:
yeah just a couple hours ago. I'll give it another whirl though :D

Reply:
It looks like this rule is only enforced when the declaration overflows a line
https://dart-lang.github.io/linter/lints/require_trailing_commas.html